src/main/java/lotto/LottoPrize.java

```diff
@@ -0,0 +1,27 @@
+package lotto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoPrize {
+  private static final Map<Integer, Integer> prize;
+
+  static {
+    prize = new HashMap<>();
+    prize.put(3, 5000);
+    prize.put(4, 50000);
+    prize.put(5, 1500000);
+    prize.put(6, 2000000000);
```

Comment:
이런식으로 가독성을 높일 수 있습니다 :)
```suggestion
    prize.put(3, 5_000);
    prize.put(4, 50_000);
    prize.put(5, 1_500_000);
    prize.put(6, 2_000_000_000);
```